### PR TITLE
chore(vagrant): update kubernetes release tarball to v1.5.1

### LINKED ISSUE #688: kube-up.sh doesn't detect Kubernetes installation
I followed the [Booting Kubernetes Using Vagrant](https://deis.com/docs/workflow/quickstart/provider/vagrant/boot/) instructions and I hit while the box was booting up:

```
$ ./cluster/kube-up.sh
...
==> master: It looks as if you don't have a compiled version of Kubernetes.  If you
==> master: are running from a clone of the git repo, please run 'make quick-release'.
==> master: Note that this requires having Docker installed.  If you are running 
==> master: from a release tarball, something is wrong.  Look at 
==> master: http://kubernetes.io/ for information on how to contact the development team for help.
The SSH command responded with a non-zero exit status. Vagrant
assumes that this means the command failed. The output for this command
should be in the log above. Please read the output to determine what
went wrong.
```

Full output is [here](https://gist.github.com/acobster/dd9d34ba83faa9153188272994e50f14).

I can't verify whether `kubectl` is in the path or anything from within Vagrant because of the SSH issue.

I tried removing the box altogether and rerunning `./cluster/kube-up.sh`, and the issue persists.

Thanks!

### PR DESCRIPTION
**Pull request by bacongobbler**
closes #688

### CODE REVIEW

File: src/quickstart/provider/vagrant/boot.md [hunk 1]
```diff
@@ -27,11 +27,10 @@ See [Kubernetes Versions](https://deis.com/docs/workflow/installing-workflow/sys
 This archive has everything that you need to launch Kubernetes. It's a fairly large archive, so it may take some time to download:
 
 ```
-$ curl -sSL https://storage.googleapis.com/kubernetes-release/release/v1.3.5/kubernetes.tar.gz -O
-$ tar -xvzf kubernetes.tar.gz
+$ curl -sSL https://storage.googleapis.com/kubernetes-release/release/v1.5.1/kubernetes.tar.gz | tar xzv
```
Comment (adibsaad) [inline src/quickstart/provider/vagrant/boot.md]: wouldn't recommend piping to shell.

Comment (bacongobbler) [inline src/quickstart/provider/vagrant/boot.md]: Could you elaborate more on that? Curious to hear your thoughts!

Comment (adibsaad) [inline src/quickstart/provider/vagrant/boot.md]: Indeed, it's common practice to pipe results of network requests to a command, but there are security concerns surrounding that. I generally try to discourage it where I can.

Comment (adibsaad) [inline src/quickstart/provider/vagrant/boot.md]: For example, there are techniques programmers can use to distinguish whether or not you're piping to shell or not and they can send different things to you if you are.

Comment (bacongobbler) [inline src/quickstart/provider/vagrant/boot.md]: ok, `curl; tar` it is then. Will that work for you?

Comment (adibsaad) [inline src/quickstart/provider/vagrant/boot.md]: Yup! Just as you had it before, where you download the file and untar separately.

Comment (bacongobbler) [inline src/quickstart/provider/vagrant/boot.md]: done.

**Pull request merged**